data/transactions/logic/ledger_test.go

```diff
@@ -190,17 +197,20 @@ func (l *Ledger) Rekey(addr basics.Address, auth basics.Address) {
 }
 
 // Round gives the current Round of the test ledger, which is random but consistent
+// Round implements interface LedgerForLogic in data/transactions/logic
 func (l *Ledger) Round() basics.Round {
 	return l.round()
 }
 
 // LatestTimestamp gives a uint64, chosen randomly.  It should
 // probably increase monotonically, but no tests care yet.
+// LatestTimestamp implements interface LedgerForLogic in data/transactions/logic
```

Comment:
Is your IDE adding all these comments?  They don't seem to add much.

Reply:
Nope. I added them as "nice-to-haves". See [this comment](https://github.com/algorand/go-algorand/pull/4322#discussion_r933511849).